src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,32 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
```

Comment:
Consider using try-with-resources for the `Session` object to ensure it is closed properly, even if an exception occurs. This would eliminate the need for the `finally` block.